### ISSUE TITLE
Need to be able to score agendas immediately

### ISSUE THREAD

**Issue by cpjolicoeur** [mrroach/CentralServer#23]
Currently, when an agenda is advanced enough times to be scored, the game only allows scoring agendas at the end of the corp turn.  Agendas need to be able to be scored immediately when they have been advanced enough.

This will allow new cards to be installed in that server, or for agendas that give tags till the end of turn (or things like that) to actually be useful to the corp

**Comment by mrroach** [comment]
Ah, there is a CorpTurnAbilities timing phase (which includes CorpScoreAgendas) after each CorpTurnActions phase, so am I right in thinking that you're specifically referring to timing phase 1.1, or is there a bigger issue?

**Comment by cpjolicoeur** [comment]
Yes, 1.1, 2.1 and post 2.2.  Maybe I clicked through 2.1 without paying attention tho
